### PR TITLE
Add Prettier for VUE

### PR DESCRIPTION
**Pull request by zfir**
## Description

Hacktoberfest PR

## Checklist

- [ ] My pull request is not a duplicate
- [ ] I added a descriptive title to this pull request
- [ ] I added a concise description or a self-explanatory screenshot to this pull request
- [ ] My code follows the code style of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -15,255 +26,266 @@
...
+      ) {
+        this.toggleNavigation();
+      }
+    },
+  },
+};
 </script>
 
 <style lang="scss">
-	@import './style/settings';
```
Comment (Abrynos) [inline src/App.vue:100]: Why did you replace all the tabs with spaces?

File: src/components/ASF/Log.vue [hunk 2]
```diff
@@ -30,150 +59,181 @@
...
 <script>
-  import { mapGetters } from 'vuex';
-  import { downloadLog } from '../../utils/download';
+import { mapGetters } from "vuex";
+import { downloadLog } from "../../utils/download";
 
-  const layoutOptions = ['time', 'process', 'level', 'logger'];
+const layoutOptions = ["time", "process", "level", "logger"];
 
-  export default {
```
Comment (Abrynos) [inline src/components/ASF/Log.vue:38]: So here you replaced the double quotes with single quotes and removed one indentation. Looks like another useless modification to me.

File: package.json [hunk 3]
```diff
@@ -9,7 +9,8 @@
     "lint": "eslint src scripts webpack --fix",
     "build": "webpack --config webpack/config.prod.js",
     "build:analyze": "webpack --config webpack/config.analyze.js",
-    "deploy": "webpack --config webpack/config.deploy.js"
+    "deploy": "webpack --config webpack/config.deploy.js",
+    "pretty-vue": "prettier --write \"**/*.vue\""
```
Comment (Abrynos) [inline package.json:13]: Oh hey. There is an actual change here.

**Review by Abrynos — COMMENTED** [review]
As far as I can see there is very few actual changes and lots of reformatting (as if you clicked the "reformat all files" button of your IDE and now want to enforce your code style on this project). Therefore I'd like to ask: What is the point of this?

**Comment by Aareksio** [comment]
@Abrynos This is what prettier does :)

@zfir Such PRs will not be accepted by this and likely any other project. Adding a linter is the lowest effort possible and doing so against the repository code style is hindering the work, not helping.

**Pull request closed without merging**